CHANGES_NEXT_RELEASE

```diff
@@ -30,3 +30,4 @@
 - Fix: Avoid rendering at the same time id and idPattern fields in NGSIv2 subscriptions
 - Fix: Error returned on GET/DELETE requests specifying Content-Type (Issue #2128)
 - Fix: Error responses with '405 Method Not Allowed' now return payload to describe the error, for v2 requests only (Issue #2075, #2078 and #2083)
+- Fix: Implemented more sofisticated HTTP Accept Header handling (Issue #1849)
```

Comment:
The five issue numbers solved by this PR should be mentioned.

In addition, not sure about "more sofisticated" is the best alternative... It is not about sofistication, but about correctness or accuracy.